Update WebView versions for api.GamepadHapticActuator.pulse

queengooborg opened the pull request:
This PR updates and corrects version values for WebView Android for the `pulse` member of the `GamepadHapticActuator` API, based upon results from the [mdn-bcd-collector](https://mdn-bcd-collector.gooborg.com) project (v7.1.2).  The collector obtains results based upon the latest WebView version (to determine if it is supported), then version numbers are copied from Chrome Android.

Tests Used: https://mdn-bcd-collector.gooborg.com/tests/api/GamepadHapticActuator/pulse

_Check out the [collector's guide on how to review this PR](https://github.com/GooborgStudios/mdn-bcd-collector#reviewing-bcd-changes)._

Elchi3 reviewed (commented):
I guess https://github.com/mdn/browser-compat-data/pull/16206 was wrong then?

queengooborg commented:
#16206 didn't actually modify `pulse`; it modified `playEffect` and `reset`!

Elchi3 commented:
https://github.com/mdn/browser-compat-data/commit/614ace132f5d5792f8488eb38a7cab0a1488c695 shows a modification to `pulse`, though.

Elchi3 reviewed (approved):
I think this API might be in Oculus. It seems like it would be very useful for VR/XR. It could therefore also be in Webview Android but I don't have these gadgets at hand right now to test. Approving but not merging yet in case you would want to test with these devices. Feel free to merge otherwise.

queengooborg commented:
> [614ace1](https://github.com/mdn/browser-compat-data/commit/614ace132f5d5792f8488eb38a7cab0a1488c695) shows a modification to `pulse`, though.

Oh what the heck?  Looks like there was a bad merge; a `secure context required` property was also changed in that commit but that's not the lines I modified...  I opened #18590 to fix this!

> I think this API might be in Oculus. It seems like it would be very useful for VR/XR. It could therefore also be in Webview Android but I don't have these gadgets at hand right now to test. Approving but not merging yet in case you would want to test with these devices. Feel free to merge otherwise.

I don't currently have access to an Android-powered XR system (including the Oculus) so I will merge this as-is!
